internal/pkg/manifest/lb_web_svc.go

```diff
@@ -4,18 +4,33 @@
 package manifest
 
 import (
+	"errors"
 	"path/filepath"
+	"time"
 
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/copilot-cli/internal/pkg/template"
 	"github.com/imdario/mergo"
+	"gopkg.in/yaml.v3"
 )
 
 const (
 	lbWebSvcManifestPath = "workloads/services/lb-web/manifest.yml"
+)
 
+// Default values for HTTPHealthCheck for a load balanced web service.
+const (
 	// LogRetentionInDays is the default log retention time in days.
-	LogRetentionInDays = 30
+	LogRetentionInDays        = 30
+	defaultHealthCheckPath    = "/"
+	defaultHealthyThreshold   = int64(2)
+	defaultUnhealthyThreshold = int64(2)
+	defaultInterval           = int64(10)
+	defaultTimeout            = int64(5)
+)
+
+var (
+	errUnmarshalHealthCheckArgs = errors.New("can't unmarshal healthcheck field into string or compose-style map")
```

Comment:
nit: let's remove words like cannot from the error message unless it's in the `cli` pkg (the root pkg)

> When adding context to returned errors, keep the context succinct by avoiding phrases like "failed to", which state the obvious and pile up as the error percolates up through the stack:

https://github.com/uber-go/guide/blob/master/style.md#error-wrapping

Reply:
I think it is fine to keep `can't` in this scenario since it is already the root error instead of wrapping the other error. Agree if we are wrapping error we want to avoid piling up like `failed to get health check: can't unmarshal health check`. And instead we want `get health check: can't unmarshal health check`. What do you all think?

Reply:
More the opposite, `cannot get health check: unmarshal health check`. Where the root error that wraps others can use "cannot" to make it more human friendly but "library" pkgs shouldn't put cannot in their error messages

Reply:
But according to https://github.com/uber-go/guide/blob/master/style.md#error-wrapping, isn't "cannot get health check: unmarshal health check" a bad example?